sample/DocumentDB-Node/index.js

```diff
@@ -0,0 +1,9 @@
+﻿module.exports = function (context, input) {
+    context.log('Node.js triggered function via EasyTables called with input ' + input);
```

Comment:
Done.

---
In reply to: [56433252](https://github.com/Azure/azure-webjobs-sdk-script/pull/112#discussion_r56433252) [](ancestors = 56433252)